vagrant-spk

```diff
@@ -676,6 +688,28 @@ def getkey(args):
                        # so we make sure to disable TTY allocation.
             )
 
+def vm_subcommand(args):
+    sandstorm_dir = os.path.join(args.work_directory, ".sandstorm")
+    if len(args.command_specific_args) == 0:
+        sys.stderr.write("Try specifying a command to pass through to vagrant, like\n\nvagrant-spk vm up\n\n")
+        sys.exit(1)
+
+    # Do a couple sanity checks.
+    ensure_host_sandstorm_folder_exists()
+    if not os.path.exists(sandstorm_dir) or not os.path.exists(os.path.join(sandstorm_dir, "Vagrantfile")):
+        sys.stderr.write("You can't call `vm` commands if there's no `.sandstorm` folder.\n" +
+                "Try `vagrant-spk setupvm <stack>` first.\n")
+        sys.exit(1)
+
+    subcommand = args.command_specific_args[0]
+    # Do subcommand specific sanity checks.
+    if subcommand == 'up':
+        # Sanity-check that the user will get a Vagrant box that supports vboxsf, since
+        # vagrant-spk depends on that for now.
+        ensure_working_vboxsf_in_base_box(sandstorm_dir)
```

Comment:
Yay. When I add another sanity-check for `up`, namely that port 6080 is available (if I can detect that the Vagrantfile uses port 6080), I'll add that here.